scripts/import-test-connections.js

```diff
@@ -282,7 +276,7 @@ async function main() {
       );
       await fs.writeFile(
         path.join(destDir, `${connectionInfoWithId.id}.json`),
-        JSON.stringify(modelWithoutSecrets)
+        JSON.stringify(connectionInfoWithoutSecrets)
```

Comment:
Shouldn’t the saved JSON objects have a single top-level `connectionInfo` key, not be the ConnectionInfo object itself?

Reply:
Yes, added in [3d6710c](https://github.com/mongodb-js/compass/pull/4647/commits/3d6710cade3deab3d9602b6ce07c32507f66dba3)